etc/config/csharp.amazon.properties

```diff
@@ -11,15 +11,18 @@ compiler.dotnet6011csharp.name=.NET 6.0.403
 compiler.dotnet6011csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.11
 compiler.dotnet6011csharp.buildConfig=Release
 compiler.dotnet6011csharp.langVersion=latest
+compiler.dotnet6011csharp.nugetPackages=/opt/compiler-explorer/dotnet-nuget
```

Comment:
We don't need to distinguish versions of dotnet-nuget, all versions of nuget packages can exists side-by-side.
Also we don't need to actually have the directory now, because the restore process won't fail for package source not existing.

Reply:
If this is an option that is the same for the whole group, you can do 
`group.csharp.nugetPackages=...` to avoid duplication. Same for fsharp and vb

Reply:
Done.